spring-mvc-basics-2/src/main/java/com/baeldung/spring/configuration/EmailConfiguration.java

```diff
@@ -53,19 +88,28 @@ public SpringTemplateEngine thymeleafTemplateEngine() {
     }
 
     @Bean
-    public SpringResourceTemplateResolver thymeleafTemplateResolver() {
-        SpringResourceTemplateResolver templateResolver = new SpringResourceTemplateResolver();
-        templateResolver.setPrefix("/WEB-INF/views/mail/");
+    public ITemplateResolver thymeleafTemplateResolver() {
+    	AbstractConfigurableTemplateResolver templateResolver;
+    	templateResolver = mailTemplatesExternal ? new FileTemplateResolver() : new ClassLoaderTemplateResolver();
```

Comment:
Declaration of the variable can be on the same line - we don't need a split declaration on L92.